### PR TITLE
Updated to core 178 and refactor from patternfly-next to patternfly.

### PR DESCRIPTION
**Pull request by dlabaj**
**What**:

Refactor from patternfly-next to patternfly to align with beta.  Updated to the latest build of core 178.

**Additional issues**:

None

### CODE REVIEW

File: packages/patternfly-4/react-core/README.md [hunk 1]
```diff
@@ -103,5 +103,5 @@ Testing is done at the root of this repo. To only run the patternfly-react tests
 yarn test packages/patternfly-4/react-core
 ```
 
-[patternfly-4]: https://github.com/patternfly/patternfly-next
```
Comment (dgutride) [inline packages/patternfly-4/react-core/README.md]: This should still be -next for now

File: packages/patternfly-4/react-styled-system/README.md [hunk 2]
```diff
@@ -135,5 +135,5 @@ Testing is done at the root of this repo. To only run the patternfly-react tests
 yarn test packages/patternfly-4/react-core
 ```
 
-[patternfly-4]: https://github.com/patternfly/patternfly-next
```
Comment (dgutride) [inline packages/patternfly-4/react-styled-system/README.md]: Still needs -next for the github repo

File: packages/patternfly-4/react-table/README.md [hunk 3]
```diff
@@ -116,5 +122,5 @@ Testing is done at the root of this repo. To only run the @patternfly/react-tabl
 yarn test packages/patternfly-4/react-table
 ```
 
-[patternfly-4]: https://github.com/patternfly/patternfly-next
```
Comment (dgutride) [inline packages/patternfly-4/react-table/README.md]: Repo name here, too

File: packages/patternfly-4/react-core/README.md (hunk 1 above)
Comment (dlabaj) [reply packages/patternfly-4/react-core/README.md]: got it thanks.

File: packages/patternfly-4/react-styled-system/README.md (hunk 2 above)
Comment (dlabaj) [reply packages/patternfly-4/react-styled-system/README.md]: Done

File: packages/patternfly-4/react-table/src/components/Table/ExpandableRowContent.js [hunk 4]
```diff
@@ -1,7 +1,7 @@
 import React from 'react';
 import PropTypes from 'prop-types';
 import { css } from '@patternfly/react-styles';
-import { tableExpandableRowContent } from '@patternfly/patternfly-next/components/Table/table.css';
+import { tableExpandableRowContent } from '@patternfly/patternfly/components/Table/table.css';
```
Comment (karelhala) [inline packages/patternfly-4/react-table/src/components/Table/ExpandableRowContent.js]: Please wait for https://github.com/patternfly/patternfly-react/pull/1314 and rebase.

Comment (dlabaj) [reply packages/patternfly-4/react-table/src/components/Table/ExpandableRowContent.js]: Updated with your changes @karelhala

File: packages/patternfly-4/react-table/README.md (hunk 3 above)
Comment (dlabaj) [reply packages/patternfly-4/react-table/README.md]: Done

**Pull request merged**